pyramid_hypernova/batch.py

```diff
@@ -157,7 +164,8 @@ def submit(self):
 
             for job_group in job_groups:
                 request_headers = self.plugin_controller.transform_request_headers({}, self.pyramid_request)
-                query = HypernovaQuery(job_group, self.batch_url, self.json_encoder, synchronous, request_headers)
+                batch_url = self.get_batch_url(job_group, self.pyramid_request)
```

Comment:
if we're being nerdy about it, is a batch is split into "groups"? should this be group_url?

Reply:
Updated the naming! this made it a bit more breaking of a change (anyone updating will need to fix their pyramid hypernova settings) but it probably helpful in terms of clarity